demo/pages/search_page/components/package_list.py

```diff
@@ -5,12 +5,18 @@
 from .package import Package
 
 
+# The ``ListComponent`` item (Package) should always be ``Component``, because
```

Comment:
Looks like this comment is not actual now (since removing of components w/o base locators)